.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "extern/lora_gateway"]
+	path = extern/lora_gateway
+	url = git@github.com:helium/lora_gateway.git
```

Comment:
This technique of fetching dependencies via Git submodules typically requires manual updates. I don't see a commit hash or tag specified so does that mean the build will just pull down the latest? I find that versioning libraries via Git submodules works fine for one or two dependencies which is likely to remain the case here.

Reply:
Since our Buildroot defconfig continues to build `libloragw` as a separate package we don't need to pin this Git submodule here.